packages/rollup-plugin-copy/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@web/rollup-plugin-copy",
+  "version": "0.0.0",
```

Comment:
Maybe start at v1?

Reply:
I wanna have at least one beta package to see if everything is alright... e.g. types, all files are included, etc.